Userland/Applications/NetworkSettings/CertStoreDialog.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Applications/NetworkSettings/CertStoreDialog.h>
+#include <Applications/NetworkSettings/CertStoreDialogGML.h>
+#include <LibCrypto/ASN1/PEM.h>
+
+namespace NetworkSettings {
+
+NonnullRefPtr<CertStoreModel> CertStoreModel::create() { return adopt_ref(*new CertStoreModel); }
+
+void CertStoreModel::load()
+{
+    // FIXME: In the future we will allow users to import own CAs, we than need to change this logic
+    auto cacert_result = Core::File::open("/etc/cacert.pem"sv, Core::File::OpenMode::Read);
+    if (cacert_result.is_error()) {
+        dbgln("Failed to load CA Certificates: {}", cacert_result.error());
+        return;
+    }
+    auto cacert_file = cacert_result.release_value();
+    auto data_result = cacert_file->read_until_eof();
+    if (data_result.is_error()) {
+        dbgln("Failed to load CA Certificates: {}", data_result.error());
+        return;
+    }
+    auto data = data_result.release_value();
+
+    auto load_result = DefaultRootCACertificates::the().reload_certificates(data);
```

Comment:
Code formatting feels a little bit awkward in this function...
I feel like this would be better formatted:
<details>
<summary><b>Expand to see code</b></summary>

```cpp
void CertStoreModel::load()
{
    // FIXME: In the future we will allow users to import own CAs, we than need to change this logic
    auto cacert_result = Core::File::open("/etc/cacert.pem"sv, Core::File::OpenMode::Read);
    if (cacert_result.is_error()) {
        dbgln("Failed to load CA Certificates: {}", cacert_result.error());
        return;
    }

    auto cacert_file = cacert_result.release_value();
    auto data_result = cacert_file->read_until_eof();
    if (data_result.is_error()) {
        dbgln("Failed to load CA Certificates: {}", data_result.error());
        return;
    }

    auto data = data_result.release_value();
    auto load_result = DefaultRootCACertificates::the().reload_certificates(data);
    if (load_result.is_error()) {
        dbgln("Failed to load CA Certificates: {}", load_result.error());
        return;
    }

    m_certificates = load_result.release_value();
}
```

</details>

It's a small change, but does make the code more readable.